README.md

```diff
@@ -131,6 +131,20 @@ calendar.business_day?(Date.parse("Sunday, 8 June 2014"))
 # => false
 ```
 
+More specifically you can check if a given `business_day?` is either a `working_day?` or a `holiday?` using methods on `Business::Calendar`.
```

Comment:
I'm wondering if people might expect `working_day?` to be an alias for `business_day?`, as sometimes we tend to use those terms interchangeably. Is it worth elaborating on it?

Reply:
Throughout the code and document it makes the distinction pretty clear e.g. in the yaml you specify "working_days" / "extra_working_days"

Reply:
e.g. the paragraph above

Reply:
Fair enough 👍